### PR TITLE
Packaging for deb and rpm

### PR DESCRIPTION
**Pull request by rnijveld**
The new ploutos based workflow will only appear once it is merged on main, and then only when a manual trigger is created (via the API or via the github user interface).

### CODE REVIEW

**Review by folkertdev — APPROVED** [review]

**Comment by folkertdev** [comment]
from our ROS contact

> Based on previous discussions, I recommend a closer look at the [systemd service file](https://github.com/pendulum-project/ntpd-rs/pull/667/files#diff-380d65349b9b168efbc4624d6dca4187835dfeb88b557d8941f30ac17bf76cf2) and its AmbientCapabilities (CAP_NET_BIND_SERVICE required for server?) and Restart configurations.

File: pkg/common/ntp.toml.default [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+log-format = "full"
+
+## The peers section allows configuring peers, you may configure multiple of
+## these blocks to add more peers to your configuration.
+## Our default configuration spawns a pool of peers (by default this attempts
+## to discover 4 distinct peers).
+[[peers]]
+mode = "pool"
+addr = "ntpd-rs.pool.ntp.org"
+#addr = "2.ntpd-rs.pool.ntp.org" # For IPv6 only networks
```
Comment (davidv1992) [inline pkg/common/ntp.toml.default:16]: No longer neccessary
```suggestion
```

File: pkg/common/ntp.toml.default [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+max-peers = 4
+
+## If you have an NTS server, you can configure a peer that connects using NTS
+## by adding a configuration such as the one below
+#[[peers]]
+#mode = "nts-server"
+#addr = "localhost:123"
```
Comment (davidv1992) [inline pkg/common/ntp.toml.default:23]: only needed in edge cases, suggest we remove this
```suggestion
```

File: pkg/common/ntp.toml.default [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+#ke-addr = "localhost:4460"
+#certificate = "/path/to/cert.pem"
```
Comment (davidv1992) [inline pkg/common/ntp.toml.default:25]: Only needed in edge cases, suggest we remove this
```suggestion
```

File: pkg/common/ntp.toml.default [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+addr = "ntpd-rs.pool.ntp.org"
+#addr = "2.ntpd-rs.pool.ntp.org" # For IPv6 only networks
+max-peers = 4
+
+## If you have an NTS server, you can configure a peer that connects using NTS
+## by adding a configuration such as the one below
+#[[peers]]
+#mode = "nts-server"
+#addr = "localhost:123"
+#ke-addr = "localhost:4460"
```
Comment (davidv1992) [inline pkg/common/ntp.toml.default:24]: Suggest we not make this localhost, but instead a public server such as the ones from netnod (Note: request for permission in progress)
```suggestion
# NTS service from NETNOD: https://www.netnod.se/nts/network-time-security
#ke-addr = "nts.netnod.se"
```

File: pkg/common/ntpd-rs.service [hunk 5]
```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Network Time Service (ntpd-rs)
+Documentation=https://github.com/pendulum-project/ntpd-rs
+After=network-online.target
+Wants=network-online.target
+Conflicts=systemd-timesyncd.service ntp.service chrony.service
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/ntp-daemon
```
Comment (davidv1992) [inline pkg/common/ntpd-rs.service:10]: Missing disabling of restart
```suggestion
Restart=no
ExecStart=/usr/bin/ntp-daemon
```

**Review by davidv1992 — COMMENTED** [review]
Looks good to me, few minor points on the configuration.

**Pull request merged**